dev-haskell/concurrent-supply/concurrent-supply-0.1.8-r1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
What about fetching the new cabal file from hackage, are you open to that?

Reply:
Yeah, having new `.cabal` file in `SRC_URI` is perfectly fine. Just make sure you are linking to an immutable versioned revision of `.cabal` file instead of a live one.